### DISCUSSION TITLE
Data sources: Fishery-independent

### DISCUSSION THREAD

**Discussion by melissamonk-NOAA** [pfmc-assessments/CA_quillback_rockfish_2025#3] [General]
List of California-specific fishery independent data sources to explore

- [x] 1. CDFW ROV
- [x] ~2. PISCO~ - UPDATE pulled PISCO data through 2001; only 26 total quillback observed; samples sizes too small to consider
- [x] ~3. ReefCheck - misidentification likely high - not considering~
- [ ] ~4. Other SMURF surveys~ My recollection is that quillback doesn't show up or isnt differentiable
- [x] 5. CCFRP
- [x] 6. NMFS collections - UPDATE pulled survey data and no new biological samples compared to 2021 assessment. 
- [ ] ~7. Diana Baetscher genetic dissertation data~ Useful for settlement timing and size but not other things

**Comment by brianlangseth-NOAA** [comment]
@melissamonk-NOAA I see you uploaded the ROV data as of November 1, 2023 onto the network drive. I am not planning to analyze formally as this would be something the state would provide, however I assume it is a resource we can pull to start to look at some trends should we wish to. Does this align with your expectation?

Also, the data run through 2021. Do you know whether more ROV years will be added?

**Comment by melissamonk-NOAA** [comment]
2. PISCO: UPDATE pulled PISCO data through 2001 on 9/20/24; 
  There are only 26 total quillback observed; samples sizes too small to consider; Quillback at this size are also too hard to identify visually and look like other species in the KGB complex
